modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/upgrade-catch-all-check/LPD_48712.testjava

```diff
@@ -0,0 +1,40 @@
+		long classNameId, long classPK, long fileEntryId, boolean cdnEnabled,
+		String cdnURL, int displayDateMonth, int displayDateDay,
+		int displayDateYear, int displayDateHour, int displayDateMinute,
+		int expirationDateMonth, int expirationDateDay, int expirationDateYear,
+		int expirationDateHour, int expirationDateMinute, boolean neverExpire,
+		boolean galleryEnabled, Map<Locale, String> titleMap, String json,
+		double priority, int type, ServiceContext serviceContext) {
+
+		CPAttachmentFileEntryLocalServiceUtil.addCPAttachmentFileEntry(123, groupId, classNameId, classPK, fileEntryId, displayDateMonth, displayDateDay, displayDateYear, displayDateHour, displayDateMinute, expirationDateMonth, expirationDateDay, expirationDateYear, expirationDateHour, expirationDateMinute, neverExpire, titleMap, json, priority, type, externalReferenceCode, serviceContext);
+		CPAttachmentFileEntryLocalServiceUtil.addCPAttachmentFileEntry(externalReferenceCode, userId, groupId, classNameId, classPK, fileEntryId, false, null, displayDateMonth, displayDateDay, displayDateYear, displayDateHour, displayDateMinute, expirationDateMonth, expirationDateDay, expirationDateYear, expirationDateHour, expirationDateMinute, neverExpire, true, titleMap, json, priority, type, serviceContext);
+		CPAttachmentFileEntryServiceUtil.addCPAttachmentFileEntry(123, groupId, classNameId, classPK, fileEntryId, displayDateMonth, displayDateDay, displayDateYear, displayDateHour, displayDateMinute, expirationDateMonth, expirationDateDay, expirationDateYear, expirationDateHour, expirationDateMinute, neverExpire, titleMap, json, priority, type, serviceContext);
+		CPAttachmentFileEntryServiceUtil.addCPAttachmentFileEntry(groupId, classNameId, classPK, fileEntryId, false, null, displayDateMonth, displayDateDay, displayDateYear, displayDateHour, displayDateMinute, expirationDateMonth, expirationDateDay, expirationDateYear, expirationDateHour, expirationDateMinute, neverExpire, true, titleMap, json, priority, type, serviceContext);
+		_cpAttachmentFileEntryLocalService.addCPAttachmentFileEntry(externalReferenceCode, userId, groupId, classNameId, classPK, fileEntryId, false, null, displayDateMonth, displayDateDay, displayDateYear, displayDateHour, displayDateMinute, expirationDateMonth, expirationDateDay, expirationDateYear, expirationDateHour, expirationDateMinute, neverExpire, true, titleMap, json, priority, type, serviceContext);
+		_cpAttachmentFileEntryService.addCPAttachmentFileEntry(groupId, classNameId, classPK, fileEntryId, false, null, displayDateMonth, displayDateDay, displayDateYear, displayDateHour, displayDateMinute, expirationDateMonth, expirationDateDay, expirationDateYear, expirationDateHour, expirationDateMinute, neverExpire, true, titleMap, json, priority, type, serviceContext);
+
+		@Reference
+		private CPAttachmentFileEntryLocalService
+			_cpAttachmentFileEntryLocalService;
+
+		@Reference
+		private CPAttachmentFileEntryService _cpAttachmentFileEntryService;
```

Comment:
@regiCesar21 break the line for both or for none of them, search for the pattern in other cases.